Refunct/Refunct.asl

```diff
@@ -1,7 +1,7 @@
 state("Refunct-Win32-Shipping")
 {
-    int level  : 0x01C79B64, 0x328, 0x18, 0x2B0, 0x114, 0xA0;
-    int resets : 0x1DCF7DC;
+    int level  : 0x01C79B64, 0xD0, 0x18, 0x2B0, 0xEC, 0xA0;
+    int resets : 0x01DFF57C, 0x4, 0x0, 0x780, 0x38C, 0xA4;
```

Comment:
This path accurately points to the level address for me for all circumstances.  Can you confirm that this points to resets for you?